runtime/src/main/java/org/corfudb/protocols/wireprotocol/ClientHandshakeHandler.java

```diff
@@ -181,11 +178,10 @@ public void channelInactive(ChannelHandlerContext ctx) throws Exception {
      *
      * @param ctx channel handler context
      * @param cause exception cause
-     * @throws Exception
      */
     @Override
     public void exceptionCaught(ChannelHandlerContext ctx,
-        java.lang.Throwable cause) throws Exception {
+        java.lang.Throwable cause) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'java.lang.Throwable' due to existing implicit import 'java.lang.*'](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651466)